client/src/components/Settings/PhoneForm.js

```diff
@@ -49,39 +50,44 @@ class PhoneForm extends React.Component {
         const { classes } = this.props;
 
         return(
-            <form 
-                className={classes.container}
-                id='phoneForm'
-                onSubmit={this.handleSubmit}
-            >
-                <div>
-                    {/* Current phone number header */}
-                    <Typography variant='h6'>Current phone:</Typography>
-                    {/* Error in loading current phone number */}
-                    {this.props.error !== '' ? <p className={classes.currentValue}>{this.props.error}</p> : null }
-                    {/* Current phone number */}
-                    <p className={classes.currentValue}>{this.props.settings.phoneNumber}</p>
-                </div>
-                {/* Text field for new phone number */}
-                <TextField 
-                    name='phoneNumber'
-                    label='New phone'
-                    className={classes.textField}
-                    value={this.state.phoneNumber}
-                    onChange={this.handleChange}
-                    margin='normal'
-                />
-                {/* Button to submit new phone number */}
-                <Button 
-                    variant='contained' 
-                    color='secondary'
-                    className={classes.button}
-                    type='submit'
-                    form='phoneForm'
+            <div className={classes.phoneFormContainer}>
```

Comment:
Currently on very large monitors this looks very streched. Even when on 1024 it seems a bit streched

Reply:
I've reduced the width at large resolutions and introduced a breakpoint at a lower resolution for larger width percentage in order to address this.